lib/constants.js

```diff
@@ -0,0 +1,6 @@
+exports.PROTOCOL = {
+  TCP: 'tcp',
+  UDS: 'uds',
+  UDP: 'udp',
+  STREAM: 'stream'
```

Comment:
Please add stream to the README

Reply:
I understand now reading farther down that you haven't added this in yet as an actual option, that this would be a follow on PR or still coming

Reply:
yes, that's the plan